### PR TITLE
update Chinese README

### PR DESCRIPTION
**Pull request by OpenVINO-dev-contest**
update Chinese README

### CODE REVIEW

**Comment by OpenVINO-dev-contest** [comment]
@eaidova @zhuo-yoyowz Pls help to review

**Review by eaidova — APPROVED** [review]

File: README_cn.md [hunk 1]
```diff
@@ -26,6 +28,19 @@
...
 |[Tiny SD](notebooks/251-tiny-sd-image-generation/)<br>[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/251-tiny-sd-image-generation/251-tiny-sd-image-generation.ipynb) | 使用Tiny-SD以及OpenVINO™实现图像生成 | <img src="https://user-images.githubusercontent.com/29454499/260904650-274fc2f9-24d2-46a3-ac3d-d660ec3c9a19.png" width=300> | |
+| [ZeroScope Text-to-video synthesis](notebooks/253-zeroscope-text2video)<br> | 使用ZeroScope 和 OpenVINO™进行文字到视频合成 | A panda eating bamboo on a rock <img src="https://github.com/itrushkin/openvino_notebooks/assets/76161256/500956d5-4aac-4710-a77c-4df34bcda3be" width=300> |
```
Comment (zhuo-yoyowz) [inline README_cn.md:31]:
```suggestion
| [ZeroScope Text-to-video synthesis](notebooks/253-zeroscope-text2video)<br> | 使用ZeroScope 和 OpenVINO™进行文字到视频生成 | A panda eating bamboo on a rock <img src="https://github.com/itrushkin/openvino_notebooks/assets/76161256/500956d5-4aac-4710-a77c-4df34bcda3be" width=300> |
```

File: README_cn.md [hunk 2]
```diff
@@ -26,6 +28,19 @@
...
+| [LLM chatbot](notebooks/254-llm-chatbot)<br> | 基于OpenVINO™的大语言模型聊天机器人 |  <img src=https://user-images.githubusercontent.com/29454499/255799218-611e7189-8979-4ef5-8a80-5a75e0136b50.png width=600> |
+| [Bark Text-to-Speech](notebooks/256-bark-text-to-audio/)<br> | 使用Bark 和 OpenVINO™进行文字生成语音 | <img src=https://github.com/openvinotoolkit/openvino_notebooks/assets/29454499/9a770279-0045-480e-95f2-1a2f2d0a5115 width=300>
```
Comment (zhuo-yoyowz) [inline README_cn.md:33]:
```suggestion
| [Bark Text-to-Speech](notebooks/256-bark-text-to-audio/)<br> | 使用Bark 和 OpenVINO™进行文本到语音转换 | <img src=https://github.com/openvinotoolkit/openvino_notebooks/assets/29454499/9a770279-0045-480e-95f2-1a2f2d0a5115 width=300>
```

File: README_cn.md [hunk 3]
```diff
@@ -26,6 +28,19 @@
...
+| [Fast Segment Anything](notebooks/261-fast-segment-anything/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F261-fast-segment-anything%2F261-fast-segment-anything.ipynb)<br>[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/261-fast-segment-anything/261-fast-segment-anything.ipynb) | 使用FastSAM 和 OpenVINO™进行目标分割 | <img src="https://user-images.githubusercontent.com/26833433/248551984-d98f0f6d-7535-45d0-b380-2e1440b52ad7.jpg" width=225> |
+| [SoftVC VITS Singing Voice Conversion](notebooks/262-softvc-voice-conversion)<br> | 基于OpenVINO™实现SoftVC VITS 演唱声音转换 | |
+| [Latent Consistency Models: the next generation of Image Generation models ](notebooks/263-latent-consistency-models-image-generation)<br> | 使用潜在一致性模型 and OpenVINO™实现图像生成 |  <img src=https://user-images.githubusercontent.com/29454499/277367065-13a8f622-8ea7-4d12-b3f8-241d4499305e.png width=300> |
```
Comment (zhuo-yoyowz) [inline README_cn.md:39]:
```suggestion
| [Latent Consistency Models: the next generation of Image Generation models ](notebooks/263-latent-consistency-models-image-generation)<br> | 使用潜在一致性模型以及 OpenVINO™实现高速图像生成 |  <img src=https://user-images.githubusercontent.com/29454499/277367065-13a8f622-8ea7-4d12-b3f8-241d4499305e.png width=300> |
```

File: README_cn.md [hunk 4]
```diff
@@ -166,6 +187,24 @@ Jupyter notebooks 分为四个大类，选择一个跟你需求相关的开始
...
 |[251-tiny-sd-image-generation](notebooks/251-tiny-sd-image-generation/)<br>[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/251-tiny-sd-image-generation/251-tiny-sd-image-generation.ipynb) | 使用Tiny-SD以及OpenVINO™实现图像生成 | <img src="https://user-images.githubusercontent.com/29454499/260904650-274fc2f9-24d2-46a3-ac3d-d660ec3c9a19.png" width=225> |
+| [252-fastcomposer-image-generation](notebooks/252-fastcomposer-image-generation/)<br> | 使用ZeroScope 和 OpenVINO™进行文字到视频合成 | |
```
Comment (zhuo-yoyowz) [inline README_cn.md:190]:
```suggestion
| [252-fastcomposer-image-generation](notebooks/252-fastcomposer-image-generation/)<br> | 使用ZeroScope 和 OpenVINO™进行文字到视频生成 | |
```

File: README_cn.md [hunk 5]
```diff
@@ -166,6 +187,24 @@ Jupyter notebooks 分为四个大类，选择一个跟你需求相关的开始
...
+| [253-zeroscope-text2video](notebooks/253-zeroscope-text2video)<br> | Text-to-video synthesis with ZeroScope and OpenVINO™ | A panda eating bamboo on a rock <img src="https://github.com/itrushkin/openvino_notebooks/assets/76161256/500956d5-4aac-4710-a77c-4df34bcda3be" width=225> |
+| [254-llm-chatbot](notebooks/254-llm-chatbot)<br> | 基于OpenVINO™的大语言模型聊天机器人 |  <img src=https://user-images.githubusercontent.com/29454499/255799218-611e7189-8979-4ef5-8a80-5a75e0136b50.png width=600> |
+| [255-mms-massively-multilingual-speech](notebooks/255-mms-massively-multilingual-speech/)<br> | MMS:  使用OpenVINO™将演说技术扩展到1000种语言以上的支持 | |
+| [256-bark-text-to-audio](notebooks/256-bark-text-to-audio)<br> | 使用Bark 和 OpenVINO™进行文字生成语音 |  <img src=https://github.com/openvinotoolkit/openvino_notebooks/assets/29454499/9a770279-0045-480e-95f2-1a2f2d0a5115 width=225> |
```
Comment (zhuo-yoyowz) [inline README_cn.md:194]:
```suggestion
| [256-bark-text-to-audio](notebooks/256-bark-text-to-audio)<br> | 使用Bark 和 OpenVINO™进行文本到语音转换 |  <img src=https://github.com/openvinotoolkit/openvino_notebooks/assets/29454499/9a770279-0045-480e-95f2-1a2f2d0a5115 width=225> |
```

File: README_cn.md [hunk 6]
```diff
@@ -26,6 +28,19 @@
...
+| [BLIP-Diffusion - Subject-Driven Generation](notebooks/258-blip-diffusion-subject-generation)<br> | 使用BLIP Diffusion和 OpenVINO™实现基于主题驱动的图像生成和修改 | <img src="https://github.com/itrushkin/openvino_notebooks/assets/76161256/0ecf621f-b544-44ae-8258-8a49be704989" width=300 />
+| [DeciDiffusion](notebooks/259-decidiffusion-image-generation/)<br> | 使用DeciDiffusion 和 OpenVINO™进行图像生成 | <img src=https://github.com/huggingface/optimum-intel/assets/29454499/cd734349-9954-4656-ab96-08a903e846ef width=300> |
+| [Fast Segment Anything](notebooks/261-fast-segment-anything/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F261-fast-segment-anything%2F261-fast-segment-anything.ipynb)<br>[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/261-fast-segment-anything/261-fast-segment-anything.ipynb) | 使用FastSAM 和 OpenVINO™进行目标分割 | <img src="https://user-images.githubusercontent.com/26833433/248551984-d98f0f6d-7535-45d0-b380-2e1440b52ad7.jpg" width=225> |
```
Comment (zhuo-yoyowz) [inline README_cn.md:37]:
```suggestion
| [Fast Segment Anything](notebooks/261-fast-segment-anything/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F261-fast-segment-anything%2F261-fast-segment-anything.ipynb)<br>[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/261-fast-segment-anything/261-fast-segment-anything.ipynb) | 使用FastSAM 和 OpenVINO™进行快速目标分割 | <img src="https://user-images.githubusercontent.com/26833433/248551984-d98f0f6d-7535-45d0-b380-2e1440b52ad7.jpg" width=225> |
```

File: README_cn.md [hunk 7]
```diff
@@ -166,6 +187,24 @@ Jupyter notebooks 分为四个大类，选择一个跟你需求相关的开始
...
+| [260-pix2struct-docvqa](notebooks/260-pix2struct-docvqa)<br>[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/260-pix2struct-docvqa/260-pix2struct-docvqa.ipynb)<br> | 使用Pix2Struct 和 OpenVINO™实现基于文档视觉的问答 | <img src=https://user-images.githubusercontent.com/29454499/276283074-df7464e6-8293-4c6c-8f77-8e95d8f94c11.png width=400>
+| [261-fast-segment-anything](notebooks/261-fast-segment-anything/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F261-fast-segment-anything%2F261-fast-segment-anything.ipynb)<br>[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/261-fast-segment-anything/261-fast-segment-anything.ipynb) | 使用FastSAM 和 OpenVINO™进行目标分割 | <img src="https://user-images.githubusercontent.com/26833433/248551984-d98f0f6d-7535-45d0-b380-2e1440b52ad7.jpg" width=225> |
```
Comment (zhuo-yoyowz) [inline README_cn.md:199]:
```suggestion
| [261-fast-segment-anything](notebooks/261-fast-segment-anything/)<br>[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F261-fast-segment-anything%2F261-fast-segment-anything.ipynb)<br>[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/261-fast-segment-anything/261-fast-segment-anything.ipynb) | 使用FastSAM 和 OpenVINO™进行快速目标分割 | <img src="https://user-images.githubusercontent.com/26833433/248551984-d98f0f6d-7535-45d0-b380-2e1440b52ad7.jpg" width=225> |
```

File: README_cn.md [hunk 8]
```diff
@@ -166,6 +187,24 @@ Jupyter notebooks 分为四个大类，选择一个跟你需求相关的开始
...
+| [262-softvc-voice-conversion](notebooks/262-softvc-voice-conversion)<br> | 基于OpenVINO™实现SoftVC VITS 演唱声音转换 | |
+| [263-latent-consistency-models-image-generation](notebooks/263-latent-consistency-models-image-generation)<br> | 使用潜在一致性模型 and OpenVINO™实现图像生成 |  <img src=https://user-images.githubusercontent.com/29454499/277367065-13a8f622-8ea7-4d12-b3f8-241d4499305e.png width=225> |
```
Comment (zhuo-yoyowz) [inline README_cn.md:201]:
```suggestion
| [263-latent-consistency-models-image-generation](notebooks/263-latent-consistency-models-image-generation)<br> | 使用潜在一致性模型 and OpenVINO™实现高速图像生成 |  <img src=https://user-images.githubusercontent.com/29454499/277367065-13a8f622-8ea7-4d12-b3f8-241d4499305e.png width=225> |
```

File: README_cn.md [hunk 9]
```diff
@@ -166,6 +187,24 @@ Jupyter notebooks 分为四个大类，选择一个跟你需求相关的开始
...
+| [264-qrcode-monster](notebooks/264-qrcode-monster/)<br> | 使用ControlNet QR Code Monster 和 OpenVINO™生成创意二维码 | <img src="https://github.com/openvinotoolkit/openvino_notebooks/assets/76463150/1a5978c6-e7a0-4824-9318-a3d8f4912c47" width=225> |
+| [265-wuerstchen-image-generation](notebooks/265-wuerstchen-image-generation)<br> | 基于Würstchen 和 OpenVINO™实现文本到图像转换 | <img src="https://github.com/itrushkin/openvino_notebooks/assets/76161256/6917c558-d74c-4cc9-b81a-679ce0a299ee" width=225> |
```
Comment (zhuo-yoyowz) [inline README_cn.md:203]:
```suggestion
| [265-wuerstchen-image-generation](notebooks/265-wuerstchen-image-generation)<br> | 基于Würstchen 和 OpenVINO™实现文本到图像生成 | <img src="https://github.com/itrushkin/openvino_notebooks/assets/76161256/6917c558-d74c-4cc9-b81a-679ce0a299ee" width=225> |
```

File: README_cn.md [hunk 10]
```diff
@@ -26,6 +28,19 @@
...
+| [SoftVC VITS Singing Voice Conversion](notebooks/262-softvc-voice-conversion)<br> | 基于OpenVINO™实现SoftVC VITS 演唱声音转换 | |
+| [Latent Consistency Models: the next generation of Image Generation models ](notebooks/263-latent-consistency-models-image-generation)<br> | 使用潜在一致性模型 and OpenVINO™实现图像生成 |  <img src=https://user-images.githubusercontent.com/29454499/277367065-13a8f622-8ea7-4d12-b3f8-241d4499305e.png width=300> |
+| [QR Code Monster](notebooks/264-qrcode-monster/)<br> | 使用ControlNet QR Code Monster 和 OpenVINO™生成创意二维码 | <img src="https://github.com/openvinotoolkit/openvino_notebooks/assets/76463150/1a5978c6-e7a0-4824-9318-a3d8f4912c47" width=225> |
+| [Würstchen](notebooks/265-wuerstchen-image-generation)<br> | 基于Würstchen 和 OpenVINO™实现文本到图像转换 | <img src="https://github.com/itrushkin/openvino_notebooks/assets/76161256/6917c558-d74c-4cc9-b81a-679ce0a299ee" width=300> | |
```
Comment (zhuo-yoyowz) [inline README_cn.md:41]:
```suggestion
| [Würstchen](notebooks/265-wuerstchen-image-generation)<br> | 基于Würstchen 和 OpenVINO™实现文本到图像生成 | <img src="https://github.com/itrushkin/openvino_notebooks/assets/76161256/6917c558-d74c-4cc9-b81a-679ce0a299ee" width=300> | |
```

File: README_cn.md (hunk 5 above)
Comment (zhuo-yoyowz) [inline README_cn.md:194]:
```suggestion
| [256-bark-text-to-audio](notebooks/256-bark-text-to-audio)<br> | 使用Bark 和 OpenVINO™进行文本到语音转换 |  <img src=https://github.com/openvinotoolkit/openvino_notebooks/assets/29454499/9a770279-0045-480e-95f2-1a2f2d0a5115 width=225> |
```

**Review by zhuo-yoyowz — APPROVED** [review]

**Comment by eaidova** [comment]
@OpenVINO-dev-contest  @zhuo-yoyowz,  thank you!

**Pull request merged**